doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+Special thanks to Lukas Doktor for his work on implementing these changes.
+
+
+## Required Use of `--user` with `pbench-move/copy-results`
+
+In preparation for the forth-coming update to the Pbench Server, where the notion of a user is introduced and all result data tar balls are tracked per-user, the `pbench-move-results` and `pbench-copy-results` commands now require that the `--user` switch be provided.  This will help facilitate migrating data into the new version of the Pbench Server.
+
+
+## Support for the New HTTP PUT Method of Posting Tar Balls
+
+In preparation for the forth-coming update to the Pbench Server, support for sending data to a Pbench Server via an HTTP PUT method has been introduced. The new commands, `pbench-results-move` and `pbench-results-push`, provide that functionality.  A token must be generated for the given user via the new `pbench-generate-token` command.
+
+The new `pbench-results-move` and `pbench-results-push` will not work with currently released versions of the Pbench Server (v0.69).  Please consult with a Pbench Server administrator for when the new version will be available for testing purposes, and/or officially released.
+
+
+## Removal of the Use of SCL (Software Collections Library)
+
+For the Pbench Agent we have removed the use of the Software Collections Library (SCL) in order to provide a Python 3 environment for RHEL 7 & CentOS 7 distributions.  We now rely on the Python 3 provided with RHEL 7.9 and CentOS 7.9.
+
+As such, the minimum supported version of RHEL and CentOS is 7.9.
+
+
+## Support for `pbench-trafficgen` dropped entirely
+
+With the release of v0.71 support for `pbench-trafficgen` has been removed in its entirety. The ability to support the behavior of that benchmark is too difficult given the implementation of `pbench-trafficgen` and `pbench-run-benchmark`'s trafficgen support.
+
+Future work on supporting benchmarks will be approached by working to have the Pbench Agent integrate with separate software packages that are dedicated to running benchmarks (unlike the Pbench Agent which only provides convenience interfaces).
+
+
+## Deprecation Notices and Removals
+
+### Deprecated Bench Scripts
+
+The following `bench-scripts` have been deprecated with this release, and will be removed entirely in the next release:
+
+| Bench Script | Comments |
+| ------------ | -------- |
+| `pbench-run-benchmark` | This interface was never completed, and only duplicates existing functionality |
+| `pbench-cyclictest`    | No replacement provided |
+| `pbench-dbench`        | No replacement provided |
+| `pbench-iozone`        | Consider using `pbench-fio` instead |
+| `pbench-migrate`       | No replacement provided |
+| `pbench-netperf`       | Consider using `pbench-uperf` instead |
+
+### Other Deprecated Interfaces
+
+ * [_**DEPRECATED**_] The `pbench-cleanup` utility command is deprecated, and will be removed in a subsequent release (see PR #1828)
```

Comment:
Do we need/want the `[_**DEPRECATED**_]` prefix here?  Both the text and the section header mention it....

Also, we don't need the comma here.